WeakAurasTemplates/TriggerTemplatesData.lua

```diff
@@ -768,6 +768,11 @@ templates.class.HUNTER = {
         { spell = 201430, type = "ability", talent = 18}, -- Stampede
         { spell = 217200, type = "ability", charges = true, requiresTarget = true, overlayGlow = true}, -- Barbed Shot
         { spell = 264667, type = "ability", buff = true}, -- Primal Rage
+        { spell = 264735, type = "ability", unit = "pet", buff = true}, -- Survival of the Fittest
+        { spell = 90361, type = "ability",  unit = "pet", buff = true}, -- Spirit Mend
+        { spell = 58875, type = "ability",  unit = "pet", buff = true}, -- Spirit Walk
+        { spell = 264265, type = "ability",  unit = "pet", buff = false}, -- Spirit Shock
+
```

Comment:
remove the new line please

Reply:
buff = false => you can remove that
buff = true => these spells buffs what ? if they buff the pet you need to add unit="pet"